scss/bitstyles/ui/sidebar.stories.mdx

```diff
@@ -81,73 +81,74 @@ Uses a fullwidth dropdown to fit with no overlap.
     {`
       <div class="u-flex u-height-100vh" x-data="{ sidebarOpen: false }">
         <nav class="u-flex">
-          <div class="u-hidden o-sidebar--large u-flex-shrink-0 u-bg-grayscale-dark-2 u-padding-m-top u-flex@l u-flex-col">
+          <div class="u-hidden o-sidebar--large u-flex-shrink-0 u-bg-grayscale-light-4 u-padding-m-top u-flex@l u-flex-col">
             <img src="${logoLarge}" width="150" height="50" alt="Company logo" class="u-flex-shrink-0 u-margin-m-left u-margin-m-right u-margin-m-bottom" />
-            <ul class="u-flex-grow-1 u-flex-shrink-1 u-overflow-y-auto u-list-none u-flex u-flex-col u-items-stretch u-padding-s4-right u-padding-s4-left">
-              <li class="u-margin-s4-bottom u-flex">
-                <a href="/" class="a-button a-button--transparent u-flex-grow-1">
+            <ul class="u-flex-grow-1 u-flex-shrink-1 u-overflow-y-auto u-list-none u-flex u-flex-col u-items-start u-items-stretch u-padding-s4-right u-padding-s4-left">
+              <li class="u-margin-s2-bottom u-flex">
+                <a href="/" class="a-button a-button--transparent u-flex-grow-1 u-justify-start">
                   <svg width="16" height="16" class="a-button-nav__icon a-icon a-icon--m u-margin-s2-right u-margin-neg-s6-left" aria-hidden="true" focusable="false">
                     <use xlink:href="${icons}#icon-home"></use>
                   </svg>
                   Dashboard
                 </a>
               </li>
-              <li class="u-margin-s4-bottom u-flex">
-                <a href="/" class="a-button a-button--transparent u-flex-grow-1" aria-current="page">
-                  <svg width="16" height="16" class="a-button-nav__icon a-icon a-icon--m u-margin-s2-right u-margin-neg-s6-left" aria-hidden="true" focusable="false">
-                    <use xlink:href="${icons}#icon-filter"></use>
+              <li class="u-margin-s2-bottom u-flex">
+                <a href="/" class="a-button a-button--transparent u-flex-grow-1 u-justify-start" aria-current="page">
```

Comment:
For all of these tickets, where we have a component that we‘ve built in JS (e.g. `Badge` and `Button`), let’s use those components in places like this — for consistency & maintenance, all the usual component advantages :)